packages/gateway/src/features/registration/root-resolvers.ts

```diff
@@ -769,7 +778,12 @@ async function markEventAsRegistered(
     await updateRegistration(id, authHeader, details, event)
   }
 
-  await registerDeclaration(id, authHeader, event)
+  const comments =
+    details?.registration?.status?.[0]?.comments?.[0]?.comment ?? undefined
+  const timeLoggedMS =
+    details?.registration?.status?.[0]?.timeLoggedMS ?? undefined
```

Comment:
@Zangetsu101 when we were talking about the timeLoggedMS bug, we should assert that it exists here too. Would require changing the GraphQL input types? It's a bigger task though, so not in scope for this pull request.